components/Dialogs/src/private/MDCAlertControllerView+Private.m

```diff
@@ -129,6 +129,12 @@ - (void)addActionButtonTitle:(NSString *)actionTitle target:(id)target selector:
   buttonRect.size.height = MAX(buttonRect.size.height, MDCDialogActionButtonHeight);
   buttonRect.size.width = MAX(buttonRect.size.width, MDCDialogActionButtonMinimumWidth);
   actionButton.frame = buttonRect;
+  CGFloat verticalInset = MIN(0, -(48 - CGRectGetHeight(actionButton.bounds)) / 2);
```

Comment:
Are these automatically updated for dynamic type?

Reply:
The edge insets or the bounds of the buttons?

Reply:
the edge insets. They should become much smaller if the button gets bigger due to dynamic type

Reply:
As far as I can see Dialogs don't support dynamic type